ftw/simplelayout/mapblock/browser/configure.zcml

```diff
@@ -0,0 +1,20 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    i18n_domain="ftw.simplelayout">
+
+    <include package=".viewlets" />
+
+    <configure zcml:condition="installed collective.geo.mapwidget">
```

Comment:
`collective.geo.mapwidget` is included in [`collective.geo.bundle`](https://pypi.python.org/pypi/collective.geo.bundle).
Since this file is only loaded when `collective.geo.bundle` is installed, we can assume that `collective.geo.mapwidget` and don't need the `zcml:condition`.